test/natalie/array_test.rb

```diff
@@ -454,19 +454,244 @@
       a = []
```

Comment:
This file is getting very large.
What would be a good way to split it up?

Reply:
Just start putting new stuff in a separate file as you go. `test/natalie/array/whatever_test.rb`

Reply:
Alternatively, if you want, you can pull spec files over from https://github.com/ruby/spec like I did with `spec/core/enumerable/*`. But I know writing the tests yourself is a bit more fun in some cases, so whichever you prefer!

Reply:
I probably will at least also pull in their tests since we are missing frozen tests and other edge cases
Is there some way to run all the tests in a directory?

Reply:
> Is there some way to run all the tests in a directory?

No. :disappointed: I think our spec runner would need to be taught how to run more than one file.